pywhat/helper.py

```diff
@@ -0,0 +1,25 @@
+"""Helper utilities"""
+import json
+import os.path
+
+
+class AvailableTags():
+    def __init__(self):
+        self.tags = set()
+        path = "Data/regex.json"
+        fullpath = os.path.join(os.path.dirname(os.path.abspath(__file__)), path)
+        with open(fullpath, "r", encoding="utf-8") as myfile:
+            regexes = json.load(myfile)
```

Comment:
We load regexes in ```    def _load_regexes(self):``` already, maybe abstract that and make that function exist only in the helper file.

Then just call that function in `distribution.py`?